src/style-auro.scss

```diff
@@ -0,0 +1,246 @@
+    box-shadow: inset 0 0 0 1px var(--auro-color-border-focus-on-dark), inset 0 0 0 3px var(--auro-color-background-lightest);
+
+    // auro-button--secondary
+    &--secondary {
+      background-color: var(--auro-color-background-lighter);
+      box-shadow: inset 0 0 0 3px var(--auro-color-ui-default-on-light);
+    }
+
+    // auro-button--tertiary
+    &--tertiary {
+      background-color: var(--auro-color-background-lighter);
+      border-color: var(--auro-color-ui-default-on-light);
+      box-shadow: inset 0 0 0 3px var(--auro-color-ui-default-on-light);
+    }
+    // auro-buttonOndark
+    &Ondark {
+      background-color: var(--auro-color-ui-default-on-dark);
+      box-shadow: inset 0 0 0 3px var(--auro-color-border-focus-on-dark);
+          
+      // auro-buttonOndark--secondary
+      &--secondary {
+        background-color: var(--auro-color-base-black-opacity-15);
+        box-shadow: inset 0 0 0 3px var(--auro-color-ui-active-on-dark);
+        border: 1px solid var(--auro-color-ui-active-on-dark);
+      }
+    
+      // auro-buttonOndark--tertiary
+      &--tertiary {
+        background-color: var(--auro-color-base-black-opacity-15);
+        border-color: var(--auro-color-ui-active-on-dark);
+        box-shadow: inset 0 0 0 3px var(--auro-color-ui-active-on-dark);
+      }
+    }
+  }
+}
+
+.auro-button {
+  @include auro_transition(all, 0.15s, ease);
+  color: var(--auro-color-text-primary-on-dark);
+  cursor: pointer;
+  background-color: var(--auro-color-ui-default-on-light);
+  border: 1px solid var(--auro-color-ui-default-on-light);
+  border-radius: 6px;
+  font-family: var(--auro-font-family-default);
+  font-size: var(--auro-text-body-size-default);
+  font-weight: var(--auro-text-body-default-weight);
+  line-height: var(--auro-unitless-scale-140);
+  overflow: hidden;
+  text-overflow: ellipsis;
+  user-select: none;
+  white-space: nowrap;
+  width: 100%;
+  -webkit-tap-highlight-color: transparent;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+
+  // handle hover state auro-button
+  :host(:not(.is-touching)) & {
+    &:hover {
+      background-color: var(--auro-color-ui-hover-on-light);
+      border: 1px solid var(--auro-color-ui-hover-on-light);
+    }
+  }
+
+  @media (hover: hover) {
+    &:active, 
+    &:hover {
+      background-color: var(--auro-color-ui-hover-on-light);
+      border: 1px solid var(--auro-color-ui-hover-on-light);
+    }
+  }
+
+
+  // handle active state
+  &:active {
+    background-color: var(--auro-color-ui-active-on-light);
+    border: 1px solid var(--auro-color-ui-active-on-light);
+    transform: scale(0.95);
+  }
+
+  @include auro_breakpoint--sm {
+    min-width: 8.75rem;
+    width: auto;
+  }
+
+  // auro-button--secondary
+  &--secondary {
+    background-color: var(--auro-color-background-lightest);
+    border: 1px solid var(--auro-color-ui-default-on-light);
+    color: var(--auro-color-text-link-on-light);
+
+    :host(:not(.is-touching)) & {
```

Comment:
Given the new @media query, is this selector still needed?

Reply:
Good call. Refactored these references out.